python/hail/expr/functions.py

```diff
@@ -184,7 +189,7 @@ def cond(condition, consequent, alternate):
                           t, indices, aggregations, joins)
 
 
-def case():
+def case() -> 'hail.expr.utils.CaseBuilder':
```

Comment:
Why is the return annotation a string? I haven't seen that before.

Reply:
There's a circular import if I don't. Strings in type hints are used for forward references, and are evaluated after everything else has run. This is how you can have matrixtables return tables and tables return matrix tables.

Reply:
Cool, thanks.